server/builtin/disable.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package builtin
+
+import (
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/proxy"
+)
+
+var disableCall = apps.Call{
+	Path: pDisable,
+	Expand: &apps.Expand{
+		AdminAccessToken: apps.ExpandAll,
+	},
+}
+
+func (a *builtinApp) disableCommandBinding() apps.Binding {
+	return apps.Binding{
+		Label:       "disable",
+		Location:    "disable",
+		Hint:        "[ App ID ]",
+		Description: "Disables an App",
+		Call:        &disableCall,
+	}
+}
+
+func (a *builtinApp) disableForm(creq apps.CallRequest) apps.CallResponse {
+	return appIDForm(disableCall)
```

Comment:
This is unnecessary complex. The form is either a dynamic form with a static select of the apps, or a static form with a dynamic select of the apps. There is no need for both to be dynamic.

The least amount of changes would be to remove the form endpoint and add to the command binding something like  `Form: appIDForm(disableCall)`.

Reply:
This is applicable in many other places where we are using the appIDForm function.